src/getPrismicContent.js

```diff
@@ -1,4 +1,5 @@
-const getPrismicContent = () => {
+const getPrismicContent = (coreParams) => {
+  const opts = Object.assign({ }, coreParams)
```

Comment:
Why is this necessary? You're only reading from it, not overwriting it
If datasource is an object with nested values, this won't deeply copy it anyway